Tests/NostrSDKTests/EventCreatingTests.swift

```diff
@@ -61,4 +61,24 @@ final class EventCreatingTests: XCTestCase, EventCreating, EventVerifying {
         XCTAssertThrowsError(try recommendServerEvent(withRelayURL: inputURL,
                                                       signedBy: Keypair.test))
     }
+
+    func testCreateReactionEvent() throws {
+        let reactedEvent = try textNote(withContent: "Hello world!",
```

Comment:
A more thorough test would be to react to a threaded reply text note to ensure that all the `e` and `p` tags (and nothing else) get copied over to the reaction event, but we currently don't handle replies yet. I'll add this test case in after we get better support for replies.